docs/Getting Started/Recurring Tasks.md

```diff
@@ -198,6 +198,8 @@ When a task has multiple dates, one of them is selected as reference date based
 2. Scheduled date
 3. Start date
 
+> If the "Remove scheduled date on recurrence" setting is true, the Start Date will be chosen before the Scheduled date.
+
```

Comment:
Looking at this change in Obsidian, I don't think that this is enough to explain the behaviour, and the reason for the behaviour.

I think it would be better to show two lists - one for each of the two scenarios.

And the important thing will be to explain _why_ there needs to be a different order...

Reply:
As I read the current docs, I don't see a why. I don't see the reason why the current behaviour was chosen. So I find it a bit difficult to conceptualise what we would describe to somebody to help them better understand why this change was made.

When you add the `when done` flag, then things get more complicated. Then the logic is a bit difficult to grasp in my opinion. Now that I've gone through the code and so on, I can predict what it will do, but I'm not sure I would have understood what it does from reading the docs.

Do you have a sense of what sort of "why" explanation would make sense here?